src/plugins/discover/public/application/components/sidebar/discover_sidebar.tsx

```diff
@@ -294,14 +393,19 @@ export function DiscoverSidebar({
         responsive={false}
       >
         <EuiFlexItem grow={false}>
-          <DiscoverIndexPattern
-            config={config}
-            selectedIndexPattern={selectedIndexPattern}
-            indexPatternList={sortBy(indexPatternList, (o) => o.attributes.title)}
-            indexPatterns={indexPatterns}
-            state={state}
-            setAppState={setAppState}
-          />
+          <EuiFlexGroup direction="row" alignItems="center">
```

Comment:
This is using medium gutter width which is too much here IMHO

Discover:
<img width="380" alt="Screenshot 2021-04-13 at 11 42 41" src="https://user-images.githubusercontent.com/1508364/114532698-5f498280-9c4d-11eb-9618-b59013bbf2a3.png">

Lens:
<img width="521" alt="Screenshot 2021-04-13 at 11 43 07" src="https://user-images.githubusercontent.com/1508364/114532754-71c3bc00-9c4d-11eb-8c2b-c856468d47d1.png">